elizhahn/karaoke-stretch-api
# Knex

## What does this PR do (summary of changes)?
 - Adds Knex files and Seed files, hooks files up to database
 
## How should it be tested?
 - Add some data to the seed file, run the seed, check postgres (might have to give `me` role Superuser authentication
 
## Future Iterations:
 - Build out seed file for data
